### DISCUSSION TITLE
Loading animation never stop

### DISCUSSION THREAD

**Discussion by ahmadxgani** [filamentphp/filament#14485] [Help] [answered]
### Package

Other

### Package Version

v3.2

### How can we help you?

The loading animation never stop, but I can access the file (it indeed exists)

![image](https://github.com/user-attachments/assets/bc11da5a-e7cb-4c75-8da9-ff9271d36f10)

I'm confused what the cause as I'm not a programmer who build the website, someone build it with filament and hand over it to me (sysadmin), and I don't have contact to the developer/programmer who actually responsible for this bug

I'm willing to debug the problem but I don't have much experience with filament or web dev in general, what I have done is search string `Menunggu ukuran berkas` with `grep` and I found it on file with name `file-upload.js`, it seems not the source code but rather transpiled file that generated by some sort of bundler? I'm not sure. I can't find the source code.

the grep search output show the following output:
```
<omitted noise>
labelFileWaitingForSize:"Menunggu ukuran berkas"
<omitted noise>
```

I assume that the cause of loading animation never stop is because it always wait for the file size information?

at this point, I'm not sure what the next steps are for debugging, any help would be appreciated, thanks in advance!

**Comment by leandrocfe** [comment, marked as answer]
Did you run `php artisan storage:link` ?
Did you config the APP_URL in the .env file?

**Reply by ahmadxgani** [reply]
Yes, I have run `php artisan storage:link` and set the `APP_URL` too

**Reply by ahmadxgani** [reply]
Ah problem solved! the value of `APP_URL` is started with http, and to fix the issue, I need to use `https` instead.

my bad didn't check the network tab activity on the dev tools :)

**Reply by ahmadxgani** [reply]
@leandrocfe  thanks for the hint! :)
